test/module/irohad/torii/query_service_test.cpp

```diff
@@ -36,7 +37,7 @@ class QueryServiceTest : public ::testing::Test {
     query_factory = std::make_shared<PbQueryFactory>();
     query_response_factory = std::make_shared<PbQueryResponseFactory>();
     // any query
-    query.mutable_payload()->mutable_get_account();
+    auto val = query.mutable_payload()->mutable_get_account();
```

Comment:
why do we need it?

Reply:
This was the way to solve current issue with the test, but it will be solved in https://github.com/hyperledger/iroha/pull/995